dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
+    return [] if level_source.nil?
+
+    responses = JSON.parse(level_source.data)
```

Comment:
+cc @cpirich @breville this is going to be a consumer of the `LevelGroup` JSON serialization format.

My understanding is that this will go away and progress will be stored on each child level.  We'll need a migration plan for PLC Evaluations if this happens after we start getting real data in the tool.